docs/platform/howto/saml/setup-saml-auth0.rst

```diff
@@ -0,0 +1,80 @@
+Set up SAML with Auth0
+=========================
+
+SAML ( *Security Assertion Markup Language* ) is a standard for
+exchanging authentication and authorization data between an identity
+provider and a service provider. To read more about SAML check the :doc:`dedicated page <saml-authentication>`.
+
+The following is the procedure to setup SAML with `Auth0 <https://auth0.com/>`_.
+
+Prerequisite steps in Aiven
+-----------------------------------
+
+1. Login to the `Aiven Console <https://console.aiven.io>`_
+
+2. Under **Projects** in the top left, click the drop down arrow and
+then on **See All Accounts**
+
+3. Click on the Account you want to edit or create a new one
+
+4. Select the **Authentication** tab
+
+5. Create a new Authentication Method, call it `Auth0` (or similar), select *Method Type* to be **SAML**, and then
+choose the team to add invited people to (or leave it blank)
+
+6. Be sure to make a note of the configuration URLs (metadata URL and ACS URL)
+
+.. note::
+   At this point, the state will be ``Pending Configuration``.
+
+Setup on Auth0
+----------------
+
+1. `Register <https://auth0.com/signup>`_ for an Auth0 account or sign into `your existing Auth0 account <https://manage.auth0.com>`_ 
+
+2. Select **Applications** and then **Create Application** 
+
+3. Give your application a name (for example, "Aiven App"), choose **Regular Web Applications**, and hit **Create** 
+
+3. Once your application has been created, go to the ``Addons`` tab and enable **SAML 2 WEB APP** option
+
+4. Click on the **SAML 2 WEB APP** option to open the settings dialog
```

Comment:
Configuring the Addon needs switching between `Settings` and `Usage` tabs, it might be worth explaining which tab these settings can be found